Docs/Book/C++Examples/example8.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Modifying molecules example8.cpp
+
+#include <iostream>
+
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/MolOps.h>
+
+int main( int argc , char **argv ) {
+
+  RDKit::ROMOL_SPTR mol1( RDKit::SmilesToMol( "CCO" ) );
```

Comment:
I've now seen that this is mentioned in the actual document under memory management. 

The code is full of RDKit::ROMol* use. Is it better to use the ROMOL_SPTR instead for new code?